### PR TITLE
Parameter fix NAS-WR02Z

### PR DESCRIPTION
**Pull request by SkorpionNL**

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -3392,200 +3392,200 @@
...
+    {
+      "id": "meter_report_interval",
+      "zwave": {
+        "index": 7,
+        "size": 2,
+		"signed": false
+      },
+      "type": "number",
+      "attr": {
+        "min": 60,
```
Comment (nlrb) [inline app.json]: Minimum is 30

File: app.json [hunk 2]
```diff
@@ -3392,200 +3392,200 @@
...
+        "size": 2
+      },
+      "type": "number",
+      "attr": {
+        "min": 1,
+        "max": 1600
+      },
+      "value": 50,
+      "label": {
+        "en": "Current change [x0.01A]",
```
Comment (nlrb) [inline app.json]: Only location with square brackets, use regular ones.

File: app.json [hunk 3]
```diff
@@ -3392,200 +3392,200 @@
...
+      "zwave": {
+        "index": 8,
+        "size": 1
+      },
+      "type": "number",
+      "attr": {
+        "min": 1,
+        "max": 16
+      },
+      "value": 13,
```
Comment (nlrb) [inline app.json]: Default is 16

File: app.json [hunk 4]
```diff
@@ -3392,200 +3392,200 @@
...
+    {
+      "id": "alarm_current",
+      "zwave": {
+        "index": 10,
+        "size": 1
+      },
+      "type": "number",
+      "attr": {
+        "min": 1,
+        "max": 16
```
Comment (nlrb) [inline app.json]: Max is 15

File: app.json [hunk 5]
```diff
@@ -3392,200 +3392,200 @@
...
+      },
+      "value": 12,
```
Comment (nlrb) [inline app.json]: Default is 13

File: app.json [hunk 6]
```diff
@@ -3392,200 +3392,200 @@
...
+        "index": 5,
+        "size": 2,
+		"signed": false
+      },
+      "type": "number",
+      "attr": {
+        "min": 1,
+        "max": 32767
+      },
+      "value": 150,
```
Comment (nlrb) [inline app.json]: Default is 120

**Comment by nlrb** [comment]
Could you also add the disabling of the physical button?
``        {
          "id": "physical_button_enabled",
          "zwave": {
            "index": 2,
            "size": 1
          },
          "type": "checkbox",
          "value": true,
          "label": {
            "en": "Physical button enabled",
            "nl": "Aan-/uitknop bruikbaar"
          },
          "hint": {
            "en": "When enabled (default), the physical button on the wall plug can be used to switch the load on/off.\nParameter: 2\nDefault: enabled = checked",
            "nl": "Wanneer aan (standaard), kan de knop op de plug gebruikt worden als aan/uit schakelaar.\nParamter: 2\nStandaard: ingeschakeld = checked"
          }
        }
``

**Comment by nlrb** [comment]
For reference:
Neo_plug_v2

**Comment by mruiter** [comment]
Updates en toevoeging is ook gedaan @nlrb 
Zo te zien worden ze tegenwoordig eindelijk met een kloppende manual geleverd en niet de standaard v1 die al sinds de start erbij zat.

Hoop dat ook bij deze "nieuwe" batch de firmware bug in de plug met de negatieve waarde voor verbruik die soms langs schieten verholpen is ?

**Comment by SkorpionNL** [comment]
Ik heb 4 nieuwe plugs besteld tijdens 11-11. Hiervan heb ik er tot nu toe 2 geïnstalleerd en zover nog geen problemen met de verkeerde negatieve waarde. @mruiter

**Comment by mruiter** [comment]
Dan lijkt het alsof ze de firmware update nu ook uitrollen op de nieuwe plugs 

Verstuurd vanaf mijn iPhone

**Comment by nlrb** [comment]
Bij mij zat nog gewoon de v1 manual bij de plugs. Deze v2 manual komt ergens van internet. Ik heb ook nog de negatieve kWh waardes.

**Comment by nlrb** [comment]
BTW: vergeten te vermelden in de review, maar bij overload wordt de led nu geel. Dus https://github.com/mruiter/com.neo/blob/35f72dade030766b338d5bb7c2b15e0b497059cf/app.json#L3539 moet worden aangepast van RED, rood, Rot naar YELLOW, geel, Gelb.

**Comment by mruiter** [comment]
Thx 💋

Verstuurd vanaf mijn iPhone

**Comment by mruiter** [comment]
Not getting any energy reports in any more after the changes. Do you guys still have power usage showing ? Im running remote atm so i cant run in debug :( Could just be i need to PTP the plug if you guys are still getting energy reports.

**Comment by nlrb** [comment]
Yes, getting power reports. But remember that some default values were written to the wrong location, so you might need to reset the plug or update all params.

**Comment by mruiter** [comment]
Yeh tried to manually update them.
I don’t hope that neo messed up and has changed the settings in the firmware and not updating the device ids so that there are now two versions of the v2 plug with different setting I’d values 🙈

Awell will have to look at home when I get back after the weekend 

Verstuurd vanaf mijn iPhone

**Comment by nlrb** [comment]
Default config is not correct. Would propose to remove id 2 & id 6 in default.

**Comment by SkorpionNL** [comment]
Power reports still work. Unfortunately I was too early with my comment about not having a negative power value. At least 1 plug is showing it right now.

**Comment by mruiter** [comment]
To bad to hear about the negative reading.

My reading are also back after an hour . I guess homey core got screwed somewhere because of the sideloaded app and needed an hour before wanting to parse the reports again 🙃

Verstuurd vanaf mijn iPhone

**Comment by mruiter** [comment]
Maybee there’s a way I can filter the negative value frontje report parser in the devices.js . Maybee when I have some more time again next week or you should come up with a idea before then 😉

Verstuurd vanaf mijn iPhone

**Comment by SkorpionNL** [comment]
The value did turn normal again. Does that happen in your situation as well that negative values turn into normal ones and then turn negative again?

**Comment by mruiter** [comment]
Yes . Sometimes the plug just seems to crash in its firmware somewhere and gives 1 it to reports that strange -2blavla number and then returns to normal like nothing happens . 

The negative is always the same . There must be a way I can make something in the report parser in the driver.js that it skips that value but I’m not a node js expert and just go with the flow 😉hope to have some time the next weeks to figure out how to skip that value so it doesn’t pass a report to homey or maybee just the last passed normal value from a buffer or something 

Verstuurd vanaf mijn iPhone

**Pull request merged**